modules/builds-configuration-parameters.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * builds/build-configuration.adoc
+
+[id="builds-configuration-parameters_{context}"]
+= Build controller configuration parameters
+
+The `build.config.openshift.io/cluster` resource offers the following
+configuration parameters.
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`Build`
+|Holds cluster-wide information on how to handle builds. The canonical name is
+`cluster`.
+
+`spec`: Holds user-settable values for the build controller
+configuration.
+
+|`BuildSpec`
+|`additionalTrustedCA`: A reference to a ConfigMap containing additional CAs that
+should be trusted for image pushes and pulls during builds. The namespace for
+this ConfigMap is `openshift-config`.
+
+|`buildDefaults`
+|Controls the default information for builds.
+
+`defaultProxy`: Contains the default proxy settings for all build operations,
+including image pull or push and source download.
+
+You can override values by setting the `HTTP_PROXY`, `HTTPS_PROXY`, and
+`NO_PROXY` environment variables in the `BuildConfig` strategy.
+
+`gitProxy`: Contains the proxy settings for Git operations only. If set, this
+overrides any Proxy settings for all Git commands, such as `git clone`.
+
+Values that are not set here are inherited from DefaultProxy.
+
+`env`: A set of default environment variables that are applied to the build if
+the specified variables do not exist on the build.
+
+`imageLabels`: A list of docker labels that are applied to the resulting image.
```

Comment:
Same here. Should instances of "docker" be replaced by "container" or something similar?

Reply:
same answer i gave above :)

Reply:
maybe the best thing is to just remove docker.  "a list of labels that are applied to the resulting image".  removes the redundancy.